examples/pipeline_wavernn/main.py

```diff
@@ -270,11 +270,11 @@ def main(args):
 
     transforms = torch.nn.Sequential(
         torchaudio.transforms.Spectrogram(**melkwargs),
-        LinearToMel(
+        torchaudio.transforms.MelScale(
```

Comment:
If you have an extra bandwidth, can you add a test that verifies that `librosa.feature.melspectrogram(S=spectrogram, ...)` and `MelScale()(spectrogram)` produce the same result?

We do have a test for `MelSpectrogram` here

https://github.com/pytorch/audio/blob/35d68fddc756b0717bfcba6048fc8e2e95efb465/test/torchaudio_unittest/transforms/librosa_compatibility_test_impl.py#L62-L87

It will be nice to add one for `MelScale` as well.

Reply:
Talking about `MelSpectrogram`, how about using `MelSpectrogram` directly?

Reply:
Makes sense.
Fixed it and checked manually.